hugo-site/content/resources/manual/contract-interface.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Contract Interfaces"
+date: 2025-04-13
+draft: false
+---
+
+## Terms
+
+- [Contract State](/manual/glossary#contract-state) - data associated with a contract that can be retrieved by Applications and Delegates.
+- [Delta](/manual/glossary#delta) - Represents a modification to some state - similar to a [diff](https://en.wikipedia.org/wiki/Diff) in source code
+- [Parameters](/manual/glossary#parameters) - Data that forms part of a contract along with the WebAssembly code
+- [State Summary](/manual/glossary#state-summary) - A compact summary of a contract's state that can be used to create a delta
+
+## Interface
+
+Freenet contracts must implement the [`ContractInterface`](https://docs.rs/freenet-stdlib/latest/freenet_stdlib/prelude/trait.ContractInterface.html) trait:
+
+```rust
```

Comment:
While this should change infrequently it would still be nice to pull it directly from the repo.